internal/cmd/gocdk/internal/static/_assets/provision/runtimevar/gcpruntimeconfig.tf

```diff
@@ -0,0 +1,28 @@
+# TODO(rvangent): Add comments explaining.
+
+locals {
+  gcpruntimeconfig_url  = "gcpruntimeconfig://projects/${local.gcp_project}/configs/${google_runtimeconfig_config.config.name}/variables/${google_runtimeconfig_variable.var.name}?decoder=string"
```

Comment:
This should probably not be `local.gcp_project` here, but something referencing the `google` provider. I wasn't sure how to do that....

Reply:
Using the [`google_project` data source](https://www.terraform.io/docs/providers/google/d/google_project.html):

```terraform
data "google_project" "project" {}

locals {
  gcpruntimeconfig_url = "gcpruntimeconfig://projects/${data.google_project.project.project_id}/configs/..."
}
```

Ideally there should be only one of these data sources per Terraform module.

Reply:
Done.